### ISSUE TITLE
Unable to BulkInsertAsync with a basic Owned Entity model - Postgres

### ISSUE THREAD

**Issue by miguellira** [borisdj/EFCore.BulkExtensions#770]
I apologize up front if this turns out to be just a basic newbie question. However, I can't seem to configure `BulkInsertAsync` to work with a model that has a simple `Owned` entity.

The following repro has a model containing a single `DbSet` with an `Owned` entity. No additional configuration. I've tried all sorts of `BulkConfig` settings but can't manage to get this to work.

**Program.cs**
```c#
using EFCore.BulkExtensions;
using Microsoft.EntityFrameworkCore;

using var dbContext = new SchoolDbContext();

await dbContext.Database.EnsureCreatedAsync();

var students = new[] {
    new Student { Name = "Jack", Address = new Address { Line1 = "125 UpTheHill St." }},
    new Student { Name = "Jill", Address = new Address { Line1 = "125 UpTheHill St.", Line2 = "Floor 250" }}
};

var bulkConfig = new BulkConfig
{
    SetOutputIdentity = true,
    PropertiesToInclude = new List<string> {
        $"{nameof(Address)}.{nameof(Address.Line1)}",
        $"{nameof(Address)}.{nameof(Address.Line2)}"
    }
};

await dbContext.BulkInsertAsync(students, bulkConfig);

public class SchoolDbContext : DbContext
{
    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
    {
        optionsBuilder.UseNpgsql("Host=localhost;Port=5432;Database=school;Username=postgres;Password=postgres;Include Error Detail=true;Log Parameters=true;Application Name=EFCore.BulkExtensions-Tester")
                      .EnableDetailedErrors(true)
                      .EnableSensitiveDataLogging(true)
                      .LogTo(Console.WriteLine);
    }

    public DbSet<Student> Students => Set<Student>();
}

public class Student
{
    public int Id { get; set; }
    public string Name { get; set; } = null!;
    public Address Address { get; set; } = null!;
}

[Owned]
public class Address
{
    public string Line1 { get; set; } = null!;
    public string? Line2 { get; set; }
}
```
**Students.csproj**
```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net6.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="EFCore.BulkExtensions" Version="6.4.1" />
    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="6.0.3" />
  </ItemGroup>

</Project>
```

**Comment by miguellira** [comment]
Cloned the repo and created a quick test to step through and got a little bit further. If I set my `BulkConfig` to the following:
```C#
        var bulkConfig = new BulkConfig
        {
            SetOutputIdentity = true,
            PropertiesToInclude = new List<string> {
                "Address.Address_Line1",
                "Address.Address_Line2"
                // $"{nameof(Address)}.{nameof(Address.Line1)}",
                // $"{nameof(Address)}.{nameof(Address.Line2)}"
            }
        };
```
I avoid the `System.Collections.Generic.KeyNotFoundException`, however, the `LoadOutputEntities` method in `TableInfo.cs` generates an invalid SQL to insert from the template table:
```sql
INSERT INTO \"Students\" () (SELECT  FROM \"StudentsTemp55ab486a\") ON CONFLICT (\"Id\") DO UPDATE SET  RETURNING \"Id\";
```
I'm wondering if this has anything to do with PostgreSQL naming conventions?

**Comment by borisdj** [comment]
Postgres was latest added DB but Owned type with it are not yet supported.
Will put Enhancement tag on the issue.

**Comment by borisdj** [comment]
Should be fixed with v6.5.5.
